.gitignore

```diff
@@ -17,6 +17,7 @@ bower_components
 node_modules
 CMakeCache.txt
 CMakeFiles/
```

Comment:
these should not be checked in?

Reply:
yeah these are generated by cmake which we don't need. we only need `Makefile` for now